spec/lib/jobs/obm-control-spec.js

```diff
@@ -127,6 +128,36 @@ describe("Job.Obm.Node", function () {
             });
         });
 
+        it('should run an OBM command with nodeId specified in options', function() {
+            var node = {
+                obmSettings: [
+                    {
+                        service: 'ipmi-obm-service',
+                        config: {
+                            "user": "admin",
+                            "password": "admin",
+                            "host": "10.0.0.254"
+                        }
+                    }
+                ]
+            };
+            mockWaterline.nodes.findByIdentifier.resolves(node);
+
+            options.nodeId = job.context.target;
```

Comment:
Since you pass the `context.target` to `options.nodeId`, so you cannot make sure the calledWith argument `nodeId` is from options or context.

I would expect there will be 4 test cases here:
- options.nodeId is empty and context.target is valid, so context.target will be used.
- options.nodeId is valid and the context.target is empty, so options.nodeId will be used.
- both options.nodeId and context.target are valid, options.nodeId will take precedence.
- both options.nodeId and context.target are empty, there will be Error threw out.